NOTICE.txt

```diff
@@ -1270,6 +1270,109 @@ SOFTWARE.
 
 ---
 
+## turndown
+
+This product contains 'turndown' by Dom Christie.
+
+A html to markdown converter.
+
+* HOMEPAGE:
+  * https://github.com/domchristie/turndown
+
+* LICENSE: MIT
+
+MIT License
+
+Copyright (c) 2017 Dom Christie
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+---
+
+## turndown-plugin-gfm
```

Comment:
Just to note, we have some small differences from GFM (most notably around list endings), but that shouldn't matter too much here

Reply:
This is because it helps a lot, but we can iterate and overwrite where is needed. It is a convenient way to support more things, but we will need to iterate and fix corner cases for sure.